apt.te

```diff
@@ -25,22 +25,25 @@ files_tmp_file(apt_tmp_t)
 type apt_tmpfs_t;
 files_tmpfs_file(apt_tmpfs_t)
 
-type apt_var_cache_t alias var_cache_apt_t;
+type apt_var_cache_t;
 files_type(apt_var_cache_t)
 
-type apt_var_lib_t alias var_lib_apt_t;
+type apt_var_lib_t;
 files_type(apt_var_lib_t)
 
 type apt_var_log_t;
 logging_log_file(apt_var_log_t)
 
+type apt_unit_t;
+init_unit_file(apt_unit_t)
```

Comment:
apt has a systemd timer, apt-daily, which can update the package database and download updated packages